### ISSUE TITLE
Rikkeisoft - Fulltime Ruby on Rails Developer - Hanoi, Vietnam - FT

### ISSUE THREAD

**Issue by hnq90** [awesome-jobs/vietnam#46]
## Location
- 3rd Floor, Sudico Bldg., Me Tri St., Nam Tu Liem Dist., Hanoi
## Requirements
- Knowledge of Ruby On Rails - > at least 1 year experienced.
- Have a good understanding of HTML5, CSS3, jQuery, Ajax...
- Have strong knowledge about MVC, OOP.
- Have experience with REST.
- Have experience with Git.
- Analytics thinking and quick learning for pickup new technologies.
- Ability to read in English.
- Have experience with Linux(Ubuntu).
## Great working environment
- Working with real projects, great change to improvement skills.
- Salary negotiable once per six months.

More detail [here](http://www.vietnamworks.com/web-developer-php-net-ruby-614538-jv)
## Contact

Send your CV with some your Ruby projects to tuyendung@rikkeisoft.com

**Comment by hnq90** [comment]
Em update lại một chút nhé anh Trung:
 
### Yêu cầu
* Có kinh nghiệm làm việc với Ruby, đặc biệt là Ruby on Rails ít nhất 1 năm.
* Ưu tiên với các CV có kèm theo giới thiệu về các dự án mà bạn đã từng làm với Ruby (ROR) với nội dung như sau:
  * Tên dự án (nếu có thể).
  * Mô tả về dự án.
  * Vai trò của bạn trong dự án.
  * Vấn đề khó khăn mà bạn đã giải quyết được và cảm thấy tâm đắc nhất trong dự án đó.
  * Github của bạn (nếu có).

**Comment by hnq90** [comment]
Cty mình vẫn cần tuyển thêm dev Ruby nhé, hoặc freelancer Ruby cũng được.

**Comment by joneslee85** [comment]
@hnq90 update vào thằng cái post gốc luôn đi

**Comment by joneslee85** [comment]
We've just updated our job post guideline, can you please update this job post to comply with it? We will remove this job post after 1 week if no changes have been made. Please see new guideline at https://github.com/ruby-vietnam/job_board/blob/master/README.md

**Comment by hnq90** [comment]
@joneslee85 a Trung add giúp em label Hanoi nhé :+1:

**Comment by joneslee85** [comment]
@hnq90 awesome

**Comment by QuyNguyenVu** [comment]
Công ty có tuyển sinh viên năm cuối thực tập RoR ko ạ ?

**Comment by hnq90** [comment]
Em gửi email đến tuyendung@rikkeisoft.com để biết thêm thông tin nhé.
Sent from Outlook Mobile




On Mon, Feb 8, 2016 at 6:10 AM -0800, "Quy Nguyen Vu" <notifications@github.com> wrote:












Công ty có tuyển sinh viên năm cuối thực tập RoR ko ạ ?

**Comment by hnq90** [comment]
Bên mình vẫn tuyển nhé. Lâu lâu up lại 1 lần cho các bạn ở HN :D

**Comment by quytv** [comment]
bên anh còn tuyển freelancer Ruby không ạ?
